simplelink_lpf3/source/ti/devices/cc23x0r5/inc/hw_types.h

```diff
@@ -115,4 +115,30 @@ typedef unsigned int  __UINT32;
 // This define is present for cross-device compatibility. Do not use!
 #define HWREGB_WRITE_LRF(x) HWREGB(x)
 
+
+//*****************************************************************************
+//
+// Macros for hardware access to bit-band supported addresses via the bit-band region.
+//
+// Macros calculate the corresponding address to access in the bit-band region
+// based on the actual address of the memory/register and the bit number.
+//
+// Do NOT use these macros to access the bit-band region directly!
+//
+//*****************************************************************************
+// Bit-band access to address x bit number b using word access (32 bit)
+#define HWREGBITW(x, b)                                                       \
```

Comment:
The CC23x0 device does not support bit banding. Consider removing this and check this isn't used anywhere.

Reply:
@SML892 Done by Stoyan -> PR updated.